db/migrate/20161017111700_add_tab_selector_support.rb

```diff
@@ -6,7 +6,7 @@ def up
                default: [],
                null: false
 
-    MyModule.update_all(shown_tabs: %w(protocols results activities samples))
+    MyModule.update_all(shown_tabs: %w(overview) + Constants::WIDGET_TYPES)
```

Comment:
I don't think this change is neccesary. If in future, you add stuff to `Constants::WIDGET_TYPES`, this migration will not work as intended any more. I believe hardcoded values are better here. And, as mentioned previously, I'd remove the `overview` tab here as it's omnipresent (yeah, I just called it that :yum:).